apps/sample-app/src/components/utils.tsx

```diff
@@ -0,0 +1,88 @@
+import { teamsDarkTheme, teamsHighContrastTheme, teamsLightTheme, Theme } from '@fluentui/react-components';
+import { app, calendar, call, chat, mail, OpenSingleChatRequest } from '@microsoft/teams-js';
+
+import { AvatarItem } from './PeopleAvatars';
+export const getTheme = (themeNow: string): Theme => {
+  switch (themeNow) {
+    case 'dark':
+      return teamsDarkTheme;
+      break;
+    case 'contrast':
+      return teamsHighContrastTheme;
+      break;
+    default:
+      return teamsLightTheme;
+  }
+};
+export function appInitializationFailed(): void {
+  app.notifyFailure({
+    reason: app.FailedReason.Other,
+    message: 'App initialization failed',
+  });
+}
+
+export function getDates(): [string, string] {
```

Comment:
this is very minor, but having this return an interface like:

```
interface DateCollection {
  today: string;
  tomorrow: string;
}
```

Would be more user friendly and easier to read. I'll let you decide whether that's needed in this sample app.